src/Wizard/Services/ConnectivityService.ts

```diff
@@ -0,0 +1,32 @@
+import { Connectivity } from '../Connectivity/Connectivity';
+import { TestType } from '../Models/ConnectivityTest';
+import { inject, injectable, injectAll } from 'tsyringe';
+import { logger } from 'src/Utils';
+import { URL } from 'url';
+
+export const ConnectivityUrls = Symbol('ConnectivityUrls');
+
+@injectable()
+export class ConnectivityService {
```

Comment:
Avoid such naming. Also, you have to introduce an interface to avoid violating DI
```suggestion
export class ConnectivityAnalyzer {
```